DeanerySystem/DeanerySystem.DataAccess/Identity/IdentityUtility.cs

```diff
@@ -26,10 +33,15 @@ public void CreateAccount(DeaneryUser user, Roles role)
 		    {
 		        new PasswordValidator<DeaneryUser>()
 		    };
-		    var userManager = new UserManager<DeaneryUser>(userStore, null, hasher, validators, passwordValidators, null,
-		        null, null, null);
-			userManager.CreateAsync(user, password: "1234567890");
-			userManager.AddToRoleAsync(user, Roles.Professor.ToString());
+		    var logger = new Logger<UserManager<DeaneryUser>>(new NullLoggerFactory());
+            var userManager = new UserManager<DeaneryUser>(userStore, null, hasher, validators, passwordValidators, null,
+		        null, null, logger);
+		    user.SecurityStamp = Guid.NewGuid().ToString();
+			var createUserResult = userManager.CreateAsync(user, password: "12qwas!@QWAS").Result;
+		    if (createUserResult.Succeeded)
+		    {
+		        var addToRoleResult = userManager.AddToRoleAsync(user, Roles.Professor.ToString()).Result;
```

Comment:
I guess here instead of `Roles.Professor.ToString()` the `role.ToString()` should be placed?

Reply:
Where is role defined?

Reply:
Isn't it a function parameter?

Reply:
Done, missed it